tests/HUBViewModelRendererTests.m

```diff
@@ -108,8 +108,8 @@ - (void)testTwoSubsequentRenders
     [self waitForExpectationsWithTimeout:2 handler:nil];
 
     XCTAssertEqual([self.collectionViewLayout numberOfInvocations], 2u);
-    XCTAssertEqualObjects([self.collectionViewLayout capturedViewModelAtIndex:0], firstViewModel);
-    XCTAssertEqualObjects([self.collectionViewLayout capturedViewModelAtIndex:1], secondViewModel);
+    XCTAssertEqual([self.collectionViewLayout capturedViewModelAtIndex:0], firstViewModel);
```

Comment:
Shouldn’t we use `XCTAssertTrue([self.collectionViewLayout capturedViewModelAtIndex:0] == firstViewModel)` here as well? As well as adding an assert description stating that we really do want to check for instance equality.

Just like at https://github.com/spotify/HubFramework/pull/259/files#diff-51a93e5f57aabdf96df5027e93c75d58R62